lib/lolcommits/plugins/lol_tumblr.rb

```diff
@@ -0,0 +1,128 @@
+# -*- encoding : utf-8 -*-
+require 'yaml'
+require 'oauth'
+require 'webrick'
+require 'cgi'
+require 'tumblr_client'
+
+module Lolcommits
+  class LolTumblr < Plugin
+    TUMBLR_API_ENDPOINT    = 'https://www.tumblr.com'
+    TUMBLR_CONSUMER_KEY    = '2FtMEDpEPkxjoUdkpHh42h9wqTu9IVS7Ra0QyNZGixdCvhllN2'
+    TUMBLR_CONSUMER_SECRET = 'qWuvxgFUR2YyWKtbWOkDTMAiBEbj7ZGaNLaNQPba0PI1N4JpBs'
+
+    def run_postcapture
+      return unless valid_configuration?
+      puts 'Posting to Tumblr'
+      r = client.photo(configuration['tumblr_name'], :data => runner.main_image)
```

Comment:
I assume `r` is a tumblr response here and you're checking for a key to ensure it uploaded OK. Can I suggest you rename this to `response` and is it possible this upload could fail here? if so perhaps rescue is needed to cover that case, and show the post FAILED message..